.devops/azure-templates/setup-env.yml

```
@@ -20,6 +20,10 @@ parameters:
     type: number
     default: 10000
 
+  - name: 'npg_sdk_url'
+    type: string
+    default: 'https://stg-ta.nexigroup.com/monetaweb/resources/hfsdk.js'
```

I would not use a default value
```suggestion
    default: ' '
```